apps/filebrowser/src/filebrowser/views.py

```diff
@@ -591,7 +596,7 @@ def listdir_paged(request, path):
       # The following should probably be deprecated
       'cwd_set': True,
       'file_filter': 'any',
-      'current_dir_path': urllib_quote(path.encode('utf-8'), safe=SAFE_CHARACTERS_URI),
+      'current_dir_path': path,
```

Comment:
Curious: Same like this is require here also or this should not be affected? https://github.com/cloudera/hue/pull/3132/files#diff-b9391b8e89e775495e3d7b0a7c8a7d8a76e26dd57e6d5db3e4350dc8e3922ab5L224-L264

Reply:
I think I just missed this one. It should not be encoded, since the corresponding value in `listdir_paged` is properly fixed. But the question is if the `def listdir` function is ever used? Can you verify?